applications/SwimmingDEMApplication/test_examples/PFEMcoupledDEMexample.gid/ProjectParameters.json

```diff
@@ -45,7 +45,7 @@
     },
     "problem_process_list" : [{
         "help" : "This process applies meshing to the problem domains",
-        "kratos_module" : "KratosMultiphysics.PfemFluidDynamicsApplication",
+        "kratos_module" : "KratosMultiphysics.PfemApplication",
```

Comment:
`"kratos_module" : "KratosMultiphysics.PfemFluidDynamicsApplication",`
would be better, because the SwimmingDEM uses the fluid always.

Reply:
Ok